### PR TITLE
Add version.json for version tracking

### PR DESCRIPTION
**Pull request by stupid-2020**
Added version.json under /build directory with corresponding script to update the file.

### CODE REVIEW

**Comment by stupid-2020** [comment]
@hao-fang As I am not familiar with build steps and make such mistake.  I could follow your idea and update the Makefile to generate the version.json by modifying the update_version.sh

**Comment by hao-fang** [comment]
> @hao-fang As I am not familiar with build steps and make such mistake. I could follow your idea and update the Makefile to generate the version.json by modifying the update_version.sh

Thanks! Sorry I didn't make it clear in the original description. Let me know if you have any questions.

**Comment by stupid-2020** [comment]
@hao-fang May I know which version of isort is using?  Mine is 4.2.2 and some options are changed (--use-parentheses is not supported):

isort -rc main.py scripts/ chat/ --check-only **--multi_line=3** --trailing-comma **--force-grid-wrap** --line-width=88

And got the following result if I use the setting above:

```
ERROR: /opt/sites/patrickw/acl2020/main.py Imports are incorrectly sorted.
ERROR: /opt/sites/patrickw/acl2020/scripts/parse_calendar.py Imports are incorrectly sorted.
ERROR: /opt/sites/patrickw/acl2020/chat/make_poster_rooms.py Imports are incorrectly sorted.
```

**Comment by hao-fang** [comment]
> @hao-fang May I know which version of isort is using? Mine is 4.2.2 and some options are changed (--use-parentheses is not supported):
> 
> isort -rc main.py scripts/ chat/ --check-only **--multi_line=3** --trailing-comma **--force-grid-wrap** --line-width=88
> 
> And got the following result if I use the setting above:
> 
> ```
> ERROR: /opt/sites/patrickw/acl2020/main.py Imports are incorrectly sorted.
> ERROR: /opt/sites/patrickw/acl2020/scripts/parse_calendar.py Imports are incorrectly sorted.
> ERROR: /opt/sites/patrickw/acl2020/chat/make_poster_rooms.py Imports are incorrectly sorted.
> ```

Mine is 4.3.21. But I think your errors are probably because your local python environment is not properly configured.
Did you follow the instruction [here](https://github.com/acl-org/acl-2020-virtual-conference/blob/master/CONTRIBUTING.md) to create your local virtualenv?

File: scripts/generate_version.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import json
+import subprocess
+import sys
+from time import strftime
+
+
+def getVersionInfo():
```
Comment (hao-fang) [inline scripts/generate_version.py:7]: can you use `get_version_info`, `show_usage`, etc. to be more consistent with Python's naming convention?

File: scripts/generate_version.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    f.write(string)
+    f.close()
+
+
+if __name__ == "__main__":
+    if len(sys.argv) == 2:
+        file_path = sys.argv[1]
+    else:
+        showUsage()
+        exit()
```
Comment (hao-fang) [inline scripts/generate_version.py:36]: There are some other CI error about this.
Can you run `make` to see if you can pass all tests locally?

Comment (stupid-2020) [reply scripts/generate_version.py:36]: Sounds the new error come from pylint.  Anyway, will update according to your suggestion later.

**Pull request merged**